core/src/main/java/site/ycsb/workloads/CoreWorkload.java

```diff
@@ -747,7 +762,6 @@ public void doTransactionRead(DB db) {
   }
 
   public void doTransactionReadModifyWrite(DB db) {
-    // choose a random key
```

Comment:
removing the comments unintended?

Reply:
It is intended since in case `keychooser` is `SequentialGenerator` it is not random